play/PlayerRules/CharacterCreation/05_ArmourList.txt

```diff
@@ -165,386 +203,284 @@ Cost For Arm or Leg Armor:	2900
+is generally the size of a small backpack, and some Field
+Stabilizers that extend the field over the body. Once the
+shield's capacitor units are exhausted, the shield breaks and
+needs a while to recharge. If the user maintains their shield 
+without letting it break, they can avoid the costly recharge time.
+For more information, please see section "Shields" in the 
+Basic Rules document.
+
+	=====   LEVEL 1 REQUIREMENT   =====
+	
+---- A-Class Electron Shield ----
+Primary Module Cost:	2310
+Prim Module Hard Pts: 	1
+Shield Strength:	120
+Shield Restart Time:	1 Turn
+Recharge Rate / Turn:	50
+Fast Recharge Nanites:	14
+
+Secondary Module Cost:	810
+Sec Module Hard Pts:	1
+Secondary Module Benefit: Either provides +60 shield strength
+OR +60 recharge rate, +4 fast recharge cost per secondary.
+Decided on purchase.
+
+	=====   LEVEL 5 REQUIREMENT   =====
+	
+---- HiCap A-Class Electron Shield ----
+Primary Module Cost:	2410
+Prim Module Hard Pts: 	2
+Shield Strength:	180
+Shield Restart Time:	1 Turn
+Recharge Rate / Turn:	30
+Fast Recharge Nanites:	36
+
+Secondary Module Cost:	840
+Sec Module Hard Pts:	1
+Secondary Module Benefit: Provides +70 shield strength 
+and +10 recharge rate, +4 fast recharge cost per secondary.
+
+---- B-Class Magnetic Shield ----
+Primary Module Cost:	3600
+Prim Module Hard Pts: 	1
+Shield Strength:	100
+Shield Restart Time:	n/a; restarts at the start of every turn if depleted
+Recharge Rate / Turn:	50
+Fast Recharge Nanites:	n/a; cannot fast recharge
+
+Secondary Module Cost:	1320
+Sec Module Hard Pts:	1
+Secondary Module Benefit: Either provides +50 shield strength
+or +50 recharge rate. Decided on purchase. 
+	
+	=====   LEVEL 10 REQUIREMENT   =====
+	
+---- C-Class Self Sustained Nanite (SSN) Shield ----
+   *Blocks All Forms of Damage*
+Primary Module Cost:	4210
+Prim Module Hard Pts: 	2
+Shield Strength:	100
+Shield Restart Time:	1 Turn
+Recharge Rate / Turn:	50
+Fast Recharge Nanites:	32
+
+Secondary Module Cost:	1470
+Sec Module Hard Pts:	1
+Secondary Module Benefit: Either provides +50 shield strength 
+or +50 recharge rate, +4 fast recharge cost per secondary. 
+Decided on purchase.
 
-    Kevlar Enhanced Combat Helemt $170
-    5 DR, 40%
 
-    Carbon-Fiber Enhanced Combat Helmet $190
-    10 DR, 40%
+    =====================================
+	     CLOAKING EQUIPMENT
+    =====================================
+Ever wanted to be invisible? Please see section
+"Cloaking" at the bottom of the Basic Rules document.
+You cannot use more than 1 cloaking primary module at 
+one time.
 
-    Kevlar Flight Helmet $240
-    5 DR, 50%
-    A simple flight helmet. does not provide full seal, but can be paired with 
-    air- and space-craft external oxygen systems.
+	=====   LEVEL 1 REQUIREMENT   =====
+	
+---- Mk1 Microfoil Cloaker ----
+Primary Module Cost:	2000
+Stealth Skill Required:	10
+Prim Module Hard Pts: 	1
+Detect DC:		75
+Moving Detect DC:   	50
+Cloak Turn Bonus:	+0

```

Comment:
What is cloak turn bonus? I don't see it in the basic rules?

Reply:
ahhh, i'll remedy that. It's just a bonus to the number of turns you can stay cloaked at one time